control_plots.py

```diff
@@ -13,10 +13,10 @@
     args = parser.parse_args()
 
     dl2_file_name = '/lustre/fs21/group/cta/users/maierg/analysis/AnalysisData/uploadDL2/Paranal_20deg/gamma_onSource.S.3HB9-FD_ID0.eff-0.root'
```

Comment:
I would try to stick to the usual line length of 120 characters max. Of course its not a big deal... But probably makes sense to separate the path from the filename, or something like that to reduce the line length.

Reply:
I actually stick to 100 max characters everywhere. This specific line is not supposed to remain in the code of course, it's specific to my setup at DESY. I will soon introduce a configuration file for the code so that such input can be given by the user instead of hardcoded in the code. For now I can split it so that we can merge though.